### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by Horstexplorer**
Just some small changes in README.md and Dockerfile

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ TeamSpeak 3 SinusBot Homepage: https://sinusbot.com/.
 
 Current version:
 ```
-Sinusbot: latest beta (>=0.11.0)
+Sinusbot: latest beta (>=0.13.37)
```
Comment (galexrt) [inline README.md]: Is the `sinusbot.current.tar.bz2` 0.13.37 or higher?

**Comment by Horstexplorer** [comment]
TS3-Version 3.1.9 should work with Sinusbot too. This could be changed but it doesn't have to.

File: README.md (hunk 1 above)
Comment (Horstexplorer) [reply README.md]: Yes it is

**Comment by galexrt** [comment]
@Horstexplorer If you test that Sinusbot works with TS3 3.1.9 I'm okay with adding in this PR.

File: README.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ By using this image you accept TeamSpeak's and Sinusbot's license agreements.
 
 When you have started the Sinusbot for the first time, you need to get the password from the logs, see [Get Sinusbot Password](#get-sinusbot-password).
 
-**WARNING** There is "no way" to show the password a second time!
+**WARNING** There is "no way" to show the password a second time! You can use ```./sinusbot --override-password=newpassword``` to reset the password.
```
Comment (galexrt) [inline README.md]: Instead of the triple \` use one instead for a better looking in line code block.

**Review by galexrt — CHANGES_REQUESTED** [review]
One small comment about the README.md change

**Comment by Horstexplorer** [comment]
I checked it, TS3 3.1.9 is working with the current Sinusbot version :)
Sinusbot.current is currently providing 0.13.37-9791176 which seems to be an older build than f7e9ece but since they are using this link everywhere, there shouldn't be any noticeable differences between them.

File: README.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ TeamSpeak 3 SinusBot Homepage: https://sinusbot.com/.
 
 Current version:
 ```
-Sinusbot: latest beta (>=0.11.0)
+Sinusbot: latest beta (>=0.13.37)
 TeamSpeak: 3.1.8 (this version is required for Sinusbot)
```
Comment (galexrt) [inline README.md]: Could you update this version number too

**Review by galexrt — CHANGES_REQUESTED** [review]
One last thing, then we are ready to go

**Review by galexrt — APPROVED** [review]

**Comment by galexrt** [comment]
Thanks for the PR!

**Pull request merged**